<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fishbot Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="forwardButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>111</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward (W)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>90</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>Left (A)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>90</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>Right (D)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>111</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop (S)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="motorButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor (L)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
